app/controllers/application_controller.rb

```diff
@@ -58,7 +58,7 @@ def decorated_user
   end
 
   def after_sign_in_path_for(user)
-    stored_location_for(user) || session[:saml_request_url] || profile_path
+    stored_location_for(user) || profile_path
```

Comment:
I suspect that we are missing a spec. If an LOA1 request is made by an existing user, they should be automatically redirected to the SP after 2FA. I believe that test will fail with this change.